posthog/api/user.py

```diff
@@ -27,6 +28,10 @@
 from posthog.utils import get_js_url
 
 
+class UserAuthenticationThrottle(UserRateThrottle):
+    rate = "5/minute"
```

Comment:
Should we only limit certain endpoints? E.g. refreshing 5 times for GET requests should be OK.